docs/user_guide/datasets/acs_public_coverage.rst

```diff
@@ -0,0 +1,197 @@
+   *  - MIG
+      - Mobility status (lived here 1 year ago):
+         1. Yes, same house (nonmovers)
+         2. No, outside US and Puerto Rico
+         3. No, different house in US or Puerto Rico
+         
+   *  - MIL
+      - Military service:
+         1. Now on active duty
+         2. On active duty in the past, but not now
+         3. Only on active duty for training in Reserves/National Guard
+         4. Never served in the military
+         
+   *  - ANC
+      - Ancestry recode:
+         1. Single
+         2. Multiple
+         3. Unclassified
+         4. Not reported
+         8. Suppressed for data year 2018 for select PUMAs
+
+   *  - NATIVITY
+      - Nativity:
+         1. Native
+         2. Foreign born
+         
+   *  - DEAR
+      - Hearing difficulty:
+         1. Yes
+         2. No
+         
+   *  - DEYE
+      - Vision difficulty:
+         1. Yes
+         2. No
+         
+   *  - DREM
+      - Cognitive difficulty:
+         1. Yes
+         2. No
+         
+   *  - PINCP
+      - Total annual income per person as an integer between -19997 and 4209995 US dollars. Loss of $19998 or more is coded as -19998. Income of $4209995 or more is coded as 4209995.
+
+   *  - ESR
+      - Employment status recode:
+         1. Civilian employed, at work
+         2. Civilian employed, with a job but not at work
+         3. Unemployed
+         4. Armed forces, at work
+         5. Armed forces, with a job but not at work
+         6. Not in labor force
+         
+   *  - ST
+      - State code:
+         Please see data dictionary at `ACS PUMS documentation <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_ for the full list of state codes.  
+
+   *  - FER
+      - Gave birth to child within the past 12 months:
+         1. Yes
+         2. No
+
+   *  - RAC1P
+      - Race code
+         1. White alone
+         2. Black or African American alone
+         3. American Indian alone
+         4. Alaska Native alone
+         5. American Indian and Alaska native tribes specified; or American Indian or Alaska Native, not specified and no other races
+         6. Asian alone
+         7. Native Hawaiian and Other Pacific Islander alone
+         8. Some Other Race alone
+         9. Two or More races
+
+
+The target label is given by PUBCOV, which can be used for a binary classification task.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 7 30
+   :stub-columns: 1
+
+   *  - Column name
+      - Description
+
+   *  - PUBCOV
+      - Public health coverage, with PUBCOV == 1 if the individual has public health coverage, else 0
+
+.. topic:: References:
+
+  .. [1] Frances Ding, Moritz Hardt, John Miller, Ludwig Schmidt `"Retiring Adult: New Datasets for Fair Machine Learning" <https://arxiv.org/pdf/2108.04884.pdf>`_,
+      Advances in Neural Information Processing Systems 34, 2021.
+
+  .. [2] R. Kohavi and B. Becker. "UCI Adult Data Set." UCI Meachine Learning Repository, 5, 1996.
```

Comment:
Did you copy this from somewhere? If so, it must have that typo there, too.
```suggestion
  .. [2] R. Kohavi and B. Becker. "UCI Adult Data Set." UCI Machine Learning Repository, 5, 1996.
```

Reply:
Ah, thanks for catching that! @kspieks Think the typo must also be in ACSIncome too then

Reply:
This typo was in my PR at some point because my text editor wasn't checking spelling and I was typing quickly 😬 It's been fixed in my PR now